manifests/l/LANCommander/Server/1.1.5/LANCommander.Server.locale.en-US.yaml

```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.9.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.9.0.schema.json
+
+PackageIdentifier: LANCommander.Server
+PackageVersion: 1.1.5
+PackageLocale: en-US
+Publisher: LANCommander
+PackageName: LANCommander Server
+License: MIT
+ShortDescription: LANCommander Server Setup

```

Comment:
Please provide a better description of what this package is, what it does, or what it is used for